### PR TITLE
Search filter/fix

### PR DESCRIPTION
**Pull request by asogwachiuba**
Fixed the explore home stacked widget
Event Filter is fully functional

### CODE REVIEW

File: lib/Widget/ibloov_appbar.dart [hunk 1]
```diff
@@ -2,13 +2,171 @@ import 'package:flutter/material.dart';
 
 import '../core/constants/colorList.dart';
 import '../gen/assets.gen.dart';
+import 'gamification_icon.dart';
 
-PreferredSizeWidget? ibloovAppbar() {
+PreferredSizeWidget? ibloovAppbar({required BuildContext context}) {
```
Comment (divincoder) [inline lib/Widget/ibloov_appbar.dart:7]: What is the context for?

File: lib/features/events/views/explore/explore_events_view.dart [hunk 2]
```diff
@@ -54,7 +54,7 @@ class ExploreEventsView extends StackedView<ExploreEventViewModel> {
                               SizedBox(height: 30.h),
                               Visibility(
                                 visible: viewModel.exploreEventData
-                                        ?.featuredEvents?.isNotEmpty ??
+                                        ?.recommendedForYou?.isNotEmpty ??
```
Comment (divincoder) [inline lib/features/events/views/explore/explore_events_view.dart:57]: Did you change this to the recommended for test purposes. You might have to change it back.

File: lib/features/events/views/explore/explore_events_view.dart [hunk 3]
```diff
@@ -64,32 +64,33 @@ class ExploreEventsView extends StackedView<ExploreEventViewModel> {
...
                               Visibility(
                                   visible: viewModel.exploreEventData
-                                          ?.featuredEvents?.isNotEmpty ??
+                                          ?.recommendedForYou?.isNotEmpty ??
                                       false,
                                   child: SizedBox(height: 10.h)),
                               Visibility(
                                 visible: viewModel.exploreEventData
-                                        ?.featuredEvents?.isNotEmpty ??
+                                        ?.recommendedForYou?.isNotEmpty ??
```
Comment (divincoder) [inline lib/features/events/views/explore/explore_events_view.dart:72]: and here

File: lib/features/events/views/explore/item/event_item_view_model.dart [hunk 4]
```diff
@@ -94,7 +96,7 @@ class EventItemViewModel extends IbloovViewModel {
     // return DateFormat('dd MMM').format(DateTime.tryParse(ticket.salesEnd!)!);
     final salesEndDate = DateTime.tryParse(ticket.salesEnd ?? "");
     if (salesEndDate != null) {
-      return daysBetween(DateTime.now(), salesEndDate);
+      return DateFormatter.daysBetween(DateTime.now(), salesEndDate);
```
Comment (divincoder) [inline lib/features/events/views/explore/item/event_item_view_model.dart:99]: we should start using extension functions for this  things soon.
we can make this change later

**Review by divincoder — APPROVED** [review]
Address the comments and merge

File: lib/features/events/views/explore/item/event_item_view_model.dart (hunk 4 above)
Comment (asogwachiuba) [reply lib/features/events/views/explore/item/event_item_view_model.dart:99]: You mean creating an extension function on the datetime object that will be doing these datatime manipulations

**Pull request merged**